frontend/public/components/build-config.tsx

```diff
@@ -126,11 +154,11 @@ const tableColumnClasses = [
 ];
 
 const displayDurationInWords = (start: string, stop: string): string => {
-  if (!start || !stop) {
+  if (!start) {
     return '-';
   }
   const startTime = new Date(start).getTime();
-  const stopTime = new Date(stop).getTime();
+  const stopTime = stop ? new Date(stop).getTime() : new Date().getTime();
   let duration = (stopTime - startTime) / 1000;
```

Comment:
I expect a round is needed here, but I'm not sure. I just saw some sub-seconds while running a BuildConfig. It's fine when it's finished.

![image](https://github.com/openshift/console/assets/139310/3967f60f-c08e-474b-8e94-a289ad35ce51)


```suggestion
  let duration = Math.round((stopTime - startTime) / 1000);
```

Reply:
Also when this wasn't part of the AC. The PipelineRun details page shows a duration while running while the Build and Shipwright BuildRun doesn't show a duration while running. Do you think we can show in on this two details pages as well? The value is already there. I think there is just another check that hides it.